en/docs/administer/product-security/logins-and-passwords/working-with-encrypted-passwords.md

```diff
@@ -191,10 +210,14 @@ To change any password which we have encrypted already, follow the below steps:
     * On Linux: `./wso2server.sh`
     * On Windows: `./wso2server.bat`
     
-2.  When you run the startup script, the following message will be prompted before starting the server: `[Enter KeyStore and Private Key Password:]`. When this is prompted, the administrator starting the server must provide the private key and keystore passwords using the command-line to proceed. Note that passwords are hidden from the terminal and log files.
+    When you run the startup script, the following message will be prompted before starting the server: `[Enter KeyStore and Private Key Password:]`.
+
+2.  When prompted, you as the administrator who is starting the server must provide the private key and keystore passwords using the command-line to proceed. 
 
-    !!!Note
-        During the server startup, it tries to connect to default user store. In order to connect to the default user store, the encrypted passwords should be decrypted. Therefore, the server admin will be prompted with key store password in order to proceed with the decryption.
+     Note that passwords are hidden from the terminal and log files.
+
+    !!! Note
```

Comment:
shouldn't this be info instead of a note?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/789/commits/9ba49a0403a2a0d620b949ed2475a8cb46a66b87